frontend/src/components/organisms/Pagination/index.jsx

```diff
@@ -1,6 +1,7 @@
 import { Pagination, PaginationVariant } from "@patternfly/react-core";
 import {
   checkTableData,
+  fetchNextJobs,
```

Comment:
I suspect all the changes to this file are erroneous merges from the old code base. 😦